lib/taskana-core/src/main/java/pro/taskana/impl/TaskServiceImpl.java

```diff
@@ -63,9 +63,19 @@
  */
 public class TaskServiceImpl implements TaskService {
 
+    private static final String IS_ALREADY_CLAIMED_BY = " is already claimed by ";
```

Comment:
Just because I'm curious: is this still a good practice or is it "old style" because the Java compiler needed it in earlier versions?

Reply:
It is still a best practice to replace literals that are used in several places by a constant. By the way, this change (as all refactoring changes I made) was requested by SonarQube...